CONTRIBUTIONS.md

```diff
@@ -0,0 +1,12 @@
+# installation
+to install we need python3.9. Run 
+```
+make setup
+```
+to install python3.9 on mac. 
+
+Then, run 
+```
+make build
+```
```

Comment:
Please add a command to active the venv as well
Other than that LGTM
I approving it, please make the necessary change.